Feat/course tests

fernandopiniani opened the pull request:
## Description

Create tests for `Course` component

## Related Issues

https://github.com/wayfair/awesome-learning/issues/105

SheaBelsky commented:
@fernandopiniani Can you pull from master to get the latest code? We fixed an error that is blocking your pipeline.

fernandopiniani commented:
> @fernandopiniani Can you pull from master to get the latest code? We fixed an error that is blocking your pipeline.

My branch is now updated. But I'm confused with this feature `setLastLessonVisited` on Link component.

After that feature, the `Link` component depends upon the `LastLessonProvider`... so should I include this provider in my unit tests or should I make the `Link` component work indepent of this provider feature?

Anyway, I made the Link component independent of the Provider. You can check it out at `src/components/Link/Link.js` changes.

SheaBelsky commented:
I think what you have is good! Thank you for taking care of this @fernandopiniani !
